src/vmm/src/guest_config/aarch64/mod.rs

```diff
@@ -1,13 +1,90 @@
 // Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-use std::collections::HashMap;
+/// Module with CPU templates for aarch64
+pub mod static_cpu_templates;
```

Comment:
`static_cpu_templates` modules is not available yet in commit b243409c47a3b009cb8016a075c87b98985392a1, introduced in the next commit bcefe2c50fc6d4e3a5284c4b6612ac151d2f9061.